ooni/nettest.py

```diff
@@ -189,21 +194,22 @@ def __init__(self, options, test_file=None, test_string=None,
 
     def getTestDetails(self):
         return {
-            'probe_asn': config.probe_ip.geodata['asn'],
-            'probe_cc': config.probe_ip.geodata['countrycode'],
-            'probe_ip': config.probe_ip.geodata['ip'],
-            'probe_city': config.probe_ip.geodata['city'],
+            'probe_asn': probe_ip.geodata['asn'],
+            'probe_cc': probe_ip.geodata['countrycode'],
+            'probe_ip': probe_ip.geodata['ip'],
+            'probe_city': probe_ip.geodata['city'],
             'software_name': 'ooniprobe',
             'software_version': ooniprobe_version,
+            # XXX only sanitize the input files
```

Comment:
Yeah this is a note for something that needs to be done, that is not currently done. Basically we treat all options as if they were filepaths and sanitize them in the sense that we strip the prefix to avoid leaking filesystem paths of the user. This can have unintended consequences if the option looks like a filepath, but actually isn't leading to it being stripped.